src/components/BlogPostCard.vue

```diff
@@ -0,0 +1,173 @@
+<template>
+  <div class="card-container">
+    <img :src="user.image" alt="Post User's Profile Picture" class="avatar-img" />
+    <div class="post-content-container">
+      <div class="user-header-container">
+        <p class="user-name-text">{{ user.firstName }} {{ user.lastName }}</p>
+        <p class="user-email-text">&centerdot; {{ user.email }}</p>
+      </div>
+      <p class="post-content-text">{{ props.post.body }}</p>
+      <p class="tags-text" v-if="props.post?.tags?.length">
```

Comment:
It would be best to put this optional chaining within a getter in the template. To avoid webpack compile issues as well as best practice to leave logic out of the template

Reply:
As well as null coalesce (??) and ternary operators (? : ) - Avoid them in the template